treeherder/config/settings.py

```diff
@@ -436,3 +443,10 @@
 
 # Resource count to limit the number of threads opening connections with the DB
 CONN_RESOURCES = 50
+
+
+#  Default environment variables check
+REQUIRED_PRODUCTION_ENV_VARIABLES = ["TREEHERDER_DJANGO_SECRET_KEY", "BROKER_URL", "DATABASE_URL", "REDIS_URL", "TREEHERDER_DEBUG", "NEW_RELIC_DEVELOPER_MODE"]
+if not DEBUG:
+    for env in REQUIRED_PRODUCTION_ENV_VARIABLES:
+        assert os.environ.get(env), "{} is a required env variable for a production set up".format(env)
```

Comment:
Do the following since it will list all that it is missing:
```python
for env in REQUIRED_PRODUCTION_ENV_VARIABLES:
   if os.environ.get(env):
      REQUIRED_PRODUCTION_ENV_VARIABLES.remove(env)
if len() != 0:
   print('For a production set up please set the missing variables: {}.format(' '.join(REQUIRED_PRODUCTION_ENV_VARIABLES))
```